backend/src/packages/sections/libs/types/section-type-to-content-map.type.ts

```diff
@@ -0,0 +1,13 @@
+import { type SectionType } from '~/packages/sections/libs/enums/enums.js';
+
+import { type Footer } from './footer.type.js';
+import { type Header } from './header.type.js';
+import { type Main } from './main.type.js';
+
+type SectionTypeToContentMap = {
+  [SectionType.HEADER]: Header;
```

Comment:
```suggestion
  [SectionType.HEADER]: SiteHeaderContent;
```
change for all
i think we need to move it to the shared module since we will need them on the frontend